.github/workflows/gh-actions.yml

```diff
@@ -9,144 +9,116 @@ jobs:
   Checks:
     runs-on: ubuntu-18.04
     steps:
-      - uses: actions/checkout@v1
-      - name: Updating packages
-        run: sudo apt-get update
-      - name: Initializtaion
-        run: sudo apt-get install doxygen vera++ cppcheck pylint pylint python-serial -q
-      - name: Signed-off
-        run: $RUNNER --check-signed-off=gh-actions
-      - name: Doxygen
-        run: $RUNNER --check-doxygen
-      - name: Vera++
-        run: $RUNNER --check-vera
-      - name: License
-        run: $RUNNER --check-license
-      - name: Magic strings
-        run: $RUNNER --check-magic-strings
-      - name: Pylint
-        run: $RUNNER --check-pylint
-      - name: Cppcheck
-        run: $RUNNER --check-cppcheck
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - run: sudo apt update
+      - run: sudo apt install doxygen vera++ cppcheck pylint python-serial
+      - run: $RUNNER --check-signed-off=gh-actions
+      - run: $RUNNER --check-doxygen
+      - run: $RUNNER --check-vera
+      - run: $RUNNER --check-license
+      - run: $RUNNER --check-magic-strings
+      - run: $RUNNER --check-pylint
+      - run: $RUNNER --check-cppcheck
 
   Linux_x86-64_Build_Correctness_Debugger_Tests:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - name: Correctness - jerry-tests
-        run: $RUNNER --jerry-tests -q
-      - name: Debugger Tests
-        run: $RUNNER --jerry-debugger -q
+      - uses: actions/checkout@v2
```

Comment:
The jerry debugger is not compatible with `python-3.x` so you should enforce the usage of `2.7`